[docs] Document new scroll actions (closes #6072) (#6086)
DevExpress/testcafe: docs/articles/templates/actions/manual-scroll-warning.md

> > Important! When you perform actions, TestCafe scrolls the webpage as needed to reach elements. There is usually no need to scroll the page manually.

Author: ```suggestion
> Important! When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. There is usually no need to use the `scroll` action explicitly.
```

> > Important! When you perform actions, TestCafe scrolls the webpage as needed to reach elements. There is usually no need to scroll the page manually.

Reviewer: ```suggestion
> Important! When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. There is usually no need to use the `scroll` action.
```

> > Important! When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. There is usually no need to use the `scroll` action.

Reviewer 2: ```suggestion
> Important! When TestCafe interacts with an off-screen DOM element, it scrolls that element into view. You do not need to use the `scroll` action.
```
